contrib/check-semver-feature.sh

```diff
@@ -112,6 +112,7 @@ check_for_non_additive_cargo_features() {
         if grep -q "FAIL" "$file"; then
             echo "You have introduced non-additive cargo features"
             echo "FAIL found in $file"
+	    cat "$file"
```

Comment:
In b8ff1a3046d7a448859e64ea6e2161ff62f1d234:

The spacing here is wrong (and below). I think it is a tabs vs space thing. It shows weirdly on my local diff.

Reply:
> tabs vs space thing

crap, the holy war never ends...